content/en/agent/kubernetes/_index.md

```diff
@@ -387,6 +387,7 @@ Additional examples are available on the [Container Discover Management][14] pag
 | ----------------------------------- | ---------------------------------------------------------------------------------------------------------------- |
 | `DD_PROCESS_AGENT_CONTAINER_SOURCE` | Overrides container source auto-detection to force a single source. e.g `"docker"`, `"ecs_fargate"`, `"kubelet"` |
 | `DD_HEALTH_PORT`                    | Set this to `5555` to expose the Agent health check at port `5555`.                                              |
+| `DD_CLUSTER_NAME`                   | Declares the cluster name that appears in Datadog                                                                |
```

Comment:
```suggestion
| `DD_CLUSTER_NAME`                   | Set a custom Kubernetes cluster identifier to avoid host alias collisions. The cluster name can be up to 40 characters with the following restrictions: Lowercase letters, numbers, and hyphens only. Must start with a letter. Must end with a number or a letter.                                                             |
```